db/changes_test.go

```diff
@@ -22,6 +22,65 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Unit test for CBG-1326
+// This will need to be manually verified by checking the logs in console
+func TestLogIfChannelsDroppedFromList(t *testing.T) {
```

Comment:
As discussed, might be better to just target FilterToAvailableChannels directly with the test, as that's the main new code.